packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -157,6 +157,7 @@ class Accordion extends AutoControlledComponent<ReactProps<AccordionProps>, any>
     _.each(panels, (panel, index) => {
       const { content, title } = panel
       const active = this.isIndexActive(index)
+      const indented = title.hasOwnProperty('icon')
```

Comment:
this is a very specific detail and does not look like functionality that is generally applicable for any Accordion; can you achieve this on the theme side (`packages/react/src/themes/teams/components/Accordion/accordionStyles.ts`)